### ISSUE TITLE
SwisClient needs BulkDelete support

### ISSUE THREAD

**Issue by tdanner** [solarwinds/orionsdk-python#43]
The SWIS REST API supports deleting many objects in a single batch - see https://github.com/solarwinds/OrionSDK/wiki/REST#bulkdelete-request. But the Python client lacks a wrapper for this feature. The implementation is pretty simple:

```python
def bulkdelete(self, uris):
	self._req("POST", "BulkDelete", {'uris': uris})
```

Noticed while investigating https://thwack.solarwinds.com/message/428474.

**Comment by superadm1n** [comment]
I think this is a good idea. I have implemented it on my fork and tested it and works as expected. I added a sample script and issued a pull request.

**Comment by danjagnow** [comment]
Resolved by #45
